evil-jumps.el

```diff
@@ -255,6 +255,7 @@ POS defaults to point."
         (evil--jumps-jump idx 1)))))
 
 (defun evil--jump-forward (count)
+  (setq evil--jumps-jump-command t)
```

Comment:
Is this the only line which is not renaming? I.e. could the PR just be this line (well, renaming aside)?

Reply:
@tomdl89 

> Is this the only line which is not renaming?

Yes, it is.

> could the PR just be this line (well, renaming aside)?

Since this line is added, the  `backward` in the name becomes  misleading. I prefer to use a better name. 

I'll revert the renaming as you requested.

Reply:
Thanks for your reply. I'm not requesting that, I just wanted to understand the PR properly. I'm happy to merge with the renaming.